src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -318,6 +324,16 @@ internal override void SetDefaultSourcePath(string defaultPath)
             WriteValueToFile<string>(fileName, "DefaultSourcePath", defaultPath);
         }
 
+        /// <summary>
+        /// Allow loading assemblies by searching in GAC.
+        /// </summary>
+        /// <returns>Boolean indicating whether assemblies should be loaded from the GAC</returns>
+        internal override bool GetAllowGacLoading()
+        {
+            string fileName = Path.Combine(psHomeConfigDirectory, configFileName);
+            return ReadValueFromFile<bool>(fileName, "AllowGacLoading");
```

Comment:
Let's push the issue while we're in beta by making it default. I'd rather people complained about performance than have no one discover this feature. We can always roll it back out and do strong documentation later. 

So yeah, I say the default should be on, and then we should get customer feedback that tells us to turn it off.